react/Viewer/Readme.md

```diff
@@ -86,3 +71,45 @@ const onFileChange = (file, nextIndex) => setState({ currentFileIndex: nextIndex
   </DemoProvider>
 </div>
 ```
+
+### Using a worker for pdfjs
+
+For performance reasons, it is important to use a web worker when showing PDF files in the viewer. If you use webpack, you should add the following alias in your configuration :
+
+```patch
++ resolve: {
++   alias: {
++     'react-pdf$' : 'react-pdf/dist/entry.webpack.js'
++   }
++ }
+```
+
+By design, JS created in the build directory are only accessible (ie served by the stack) if you are logged in. It also goes for the worker file created for pdfjs. If you need it on a public page, you must tell webpack to create it in a public folder, that will be served by the stack even if the user is not logged in. You can configure which folders of your build are served publicly in your manifest.webapp.
```

Comment:
Vu avec Yannick : When aliasing react-pdf with this webpack specific line, a worker JS file will be created.